### PR TITLE
Ft show nodes

### PR DESCRIPTION
**Pull request by Mubangizi**
Depends on Pods PR

### CODE REVIEW

File: src/components/NodesList/index.jsx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+  getAge(utcDate) {
+    const creationTimestamp = new Date(utcDate).getTime();
+    return creationTimestamp;
+  }
+
+  getRoles(node) {
+    if (typeof (node.spec.taints) !== 'undefined') {
+      const str = String(node.spec.taints[0].key);
+      // get strings after /
```
Comment (neelxie) [inline src/components/NodesList/index.jsx]: Kindly explain this comment.

File: src/components/NodesList/index.jsx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    if (status === 'True') {
+      return true;
+    }
+    return false;
+  }
+
+
+  render() {
+    const { nodes, isFetched, isRetrieving } = this.props;
+    const { clusterName } = this.props;
```
Comment (neelxie) [inline src/components/NodesList/index.jsx]: This line is not working and not returning the required value, I have a fix for this in my PR.

**Comment by StephenAraka** [comment]
> Depends on Pods PR

Does this explain why there are conflicts or can the conflicts be resolved?

File: src/components/NodesList/index.jsx (hunk 1 above)
Comment (Mubangizi) [reply src/components/NodesList/index.jsx]: `str` gets a long string and am interested in the role string which is after a `/`. So the next line returns all characters after `/`

**Comment by Mubangizi** [comment]
> > Depends on Pods PR
> 
> Does this explain why there are conflicts or can the conflicts be resolved?

Pods PR has an `AgeMethod` that i need to use in this Pr, thats the only dependance i need.
As for the Conflicts those Were caused in the `develop-new` branch from merging Henrys PR

File: src/components/NodesList/index.jsx (hunk 1 above)
Comment (neelxie) [reply src/components/NodesList/index.jsx]: @Mubangizi thanks

**Comment by Mubangizi** [comment]
Everything is working perfectly
All feed back has been worked upon thank you

**Pull request merged**